sdk/aot/azure-aot-graalvm-support/src/main/java/com/azure/aot/graalvm/support/implementation/features/StorageBlobFeature.java

```diff
@@ -25,7 +25,6 @@ public Set<String[]> getDynamicProxies() {
             interfaces("com.azure.storage.blob.implementation.BlobsImpl$BlobsService"),
             interfaces("com.azure.storage.blob.implementation.BlockBlobsImpl$BlockBlobsService"),
             interfaces("com.azure.storage.blob.implementation.ContainersImpl$ContainersService"),
-            interfaces("com.azure.storage.blob.implementation.DirectoriesImpl$DirectoriesService"),
```

Comment:
why delete this line?

Reply:
This class DirectoriesImpl does not exist.